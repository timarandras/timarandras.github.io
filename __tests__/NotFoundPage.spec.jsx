import { render } from '@testing-library/react';
import { useStaticQuery } from 'gatsby';
import React from 'react';

import NotFoundPage from '../src/pages/404';

describe('NotFoundPage', () => {
  test('Matches snapshot', () => {
    useStaticQuery.mockReturnValueOnce({ site: { siteMetadata: { description: 'Dummy site' } } });
    const { container } = render(<NotFoundPage />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
