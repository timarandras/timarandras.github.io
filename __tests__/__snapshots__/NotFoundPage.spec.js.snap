// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Matches snapshot 1`] = `
<div
  class="MuiContainer-root MuiContainer-maxWidthLg"
>
  <h1>
    NOT FOUND
  </h1>
  <p>
    You just hit a route that doesn't exist... the sadness.
  </p>
</div>
`;
